compiler/one-cmds/dummy-driver/src/dummy-init.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * dummy-init only tests its interface rather than its functionality.
+ *
+ * $ ./dummy-init
+ *
+ * ... json output
+ *
+ */
+
+#include <iostream>
+
+const char *output =
+  R"(
+{
+  \"workflow\": [\"one-import\", \"one-optimize\", \"one-quantize\", \"one-codegen\"],
```

Comment:
I've searched for this `R("...")`, raw string and looks good feature.
And I think we don't need to use `\"` but just `"` seems OK.